patches/0075-qsv-use-a-new-method-to-create-mfx-session-when-usin.patch

```diff
@@ -16,9 +16,10 @@ This is in preparation for oneVPL support
 
 [1] https://spec.oneapi.io/versions/latest/elements/oneVPL/source/programming_guide/VPL_prg_session.html#onevpl-dispatcher
 
+Co-authored-by: galinart <artem.galin@intel.com>
 Signed-off-by: galinart <artem.galin@intel.com>
 ---
- libavcodec/qsv.c                 | 197 ++++++++++++--
+ libavcodec/qsv.c                 | 197 ++++++++++--
```

Comment:
do we need to update create_mfx_session in libavcodec/qsv.c?

Reply:
create_mfx_session() in libavcodec/qsv.c are used in two cases
```
$ ffmpeg -c:v h264_qsv -i ~/graphics/movies/720p.mp4 -f null -
```
There isn't hw device info because '-hwaccel qsv' isn't set, we have to use the default setting.
```
$ ffmpeg -hwaccel qsv -c:v h264_qsv -i ~/graphics/movies/720p.mp4 -f null -
```
create_mfx_session() is used to create a child session for qsv decoder/encoder. In this case, the given loader is not NULL and has been configured with right config filters in libavutil/hwcontext_qsv.c